cli/src/main/scala/com/ibm/sparktc/sparkbench/cli/Configurator.scala

```diff
@@ -21,14 +22,14 @@ object Configurator {
 
   private def parseSparkBenchRunConfig(config: Config): Seq[MultiSuiteRunConfig] = {
     val sparkContextConfs = getConfigListByName("spark-submit-config", config)
-    sparkContextConfs.map { sparkContextConf =>
-      val sparkConfs = Try(sparkContextConf.getObject("conf")).map(toStringMap).getOrElse(Map.empty)
-
-      MultiSuiteRunConfig(
-        suitesParallel = Try(sparkContextConf.getBoolean("suites-parallel")).getOrElse(false),
-        suites = getConfigListByName("suites", sparkContextConf).map(parseSuite)
-      )
+    val workloadConfs = sparkContextConfs.map { sparkContextConf => {
+        MultiSuiteRunConfig(
+          suitesParallel = Try(sparkContextConf.getBoolean("suites-parallel")).getOrElse(false),
+          suites = getConfigListByName("workload-suites", sparkContextConf).map(parseSuite)
```

Comment:
Is there a purpose behind the name change "suites" to "workload-suites"?  Is there a key "datagen-suites" as well?